concepticondata/conceptlists/Huber-1992-375.tsv

```diff
@@ -0,0 +1,369 @@
+Huber-1992-375-244	250	quebrada	stream	947	STREAM
+Huber-1992-375-245	251	piedrecillas	pebbles	961	PEBBLES
+Huber-1992-375-246	252	rocas	huge rocks		
+Huber-1992-375-247	253	camino, sendero, trocha	path	2252	PATH
+Huber-1992-375-248	255	rancho	shelter	1678	HUT
+Huber-1992-375-249	256	este	this	1214	THIS
+Huber-1992-375-250	257	eso, ese, aquel	that	78	THAT
+Huber-1992-375-251	258	quién	who	1235	WHO
+Huber-1992-375-252	259	que	what	1236	WHAT
+Huber-1992-375-253	260	no	not	1240	NOT
+Huber-1992-375-254	261	todos	all	98	ALL
+Huber-1992-375-255	262	muchos	many	1198	MANY
+Huber-1992-375-256	263	largo	long	1203	LONG
+Huber-1992-375-257	264	corteza	bark	1204	BARK
+Huber-1992-375-258	265	carne	flesh	2259	FLESH
+Huber-1992-375-259	266	sangre	blood	946	BLOOD
+Huber-1992-375-260	267	grasa	grease, animal fat	1282	FAT (FROM ANIMALS)
+Huber-1992-375-261	268	huevo	egg	744	EGG
+Huber-1992-375-262	269	cuerno	horn	1393	HORN (ANATOMY)
+Huber-1992-375-263	270	pluma	feather	1201	FEATHER
+Huber-1992-375-264	271	uña	fingernail, claw	2128	CLAW OR NAIL
+Huber-1992-375-265	272	barriga	belly	1251	BELLY
+Huber-1992-375-266	273	hígado	liver	1224	LIVER
+Huber-1992-375-267	274	beber	drink	1401	DRINK
+Huber-1992-375-268	275	comer	eat	1336	EAT
+Huber-1992-375-269	276	morder	bite	1403	BITE
+Huber-1992-375-270	277	ver	see	1409	SEE
+Huber-1992-375-271	278	oir	hear	1408	HEAR
+Huber-1992-375-272	279	saber	know	1410	KNOW (SOMETHING)
+Huber-1992-375-273	280	dormir	sleep	1585	SLEEP
+Huber-1992-375-274	281	morir	die	1494	DIE
+Huber-1992-375-275	282	matar	kill	1417	KILL
+Huber-1992-375-276	283	nadar	swim	1439	SWIM
+Huber-1992-375-277	284	volar	fly	1441	FLY (MOVE THROUGH AIR)
+Huber-1992-375-278	285	andar	walk	1443	WALK
+Huber-1992-375-279	286	venir	come	1446	COME
+Huber-1992-375-280	287	acostarse	lie down	215	LIE DOWN
+Huber-1992-375-281	288	sentarse	sit	1649	SIT DOWN
+Huber-1992-375-282	289	ponerse en pie, de pie	stand	1442	STAND
+Huber-1992-375-283	290	dar	give	1447	GIVE
+Huber-1992-375-284	291	decir	say	1458	SAY
+Huber-1992-375-285	292	quemar, arder	burn	2102	BURN
+Huber-1992-375-286	293	montaña	mountain	639	MOUNTAIN
+Huber-1992-375-287	294	rojo	red	156	RED
+Huber-1992-375-288	295	verde	green	1425	GREEN
+Huber-1992-375-289	296	amarillo	yellow	1424	YELLOW
+Huber-1992-375-290	297	lleno	full	1429	FULL
+Huber-1992-375-291	298	nuevo	new	1231	NEW
+Huber-1992-375-292	299	redondo	round	1395	ROUND
+Huber-1992-375-293	300	seco	dry	1398	DRY
+Huber-1992-375-294	301	nombre	name	1405	NAME
+Huber-1992-375-295	302	cómo	how	1239	HOW
+Huber-1992-375-296	303	cuándo	when	1238	WHEN
+Huber-1992-375-297	304	dónde	where	1237	WHERE
+Huber-1992-375-298	305	aquí	here	136	HERE
+Huber-1992-375-299	306	allí	there	1937	THERE
+Huber-1992-375-300	307	otro	other	197	OTHER
+Huber-1992-375-301	308	pocos	few	1242	FEW
+Huber-1992-375-302	309	neblina	fog	249	FOG
+Huber-1992-375-303	310	fluir	flow	2003	FLOW
+Huber-1992-375-304	311	mar	sea	1474	SEA
+Huber-1992-375-305	312	mojar	wet	1726	WET
```

Comment:
We don't have a precise verb for that, which would be "to wet (something); to dampen". But 1726 | WET is a property, at most we can use 2324 | GET WET (reflexive in Spanish).